Translation of chapters 12 and changelog to Portuguese (#479)
OWASP/owasp-masvs: Document-ptpt/CHANGELOG.md

# Registo de Alterações

## V1.2 - 7 de Março de 2020 - Lançamento da versão Internacional

As seguintes alterações fazem parte do lançamento da versão 1.2:

- Tradução do MASVS disponível em Chinês simplificado.
- Alteração do título na capa do livro MASVS.
- Removidos do MSTG, o Mobile Top 10 e CWE, e incorporados nas referências já existentes em MASVS.

## V1.2-RC - 5 de Outubro de 2019 - Pré-Lançamento de versão (apenas em Inglês)

As seguintes alterações fazem parte do pré lançamento da versão 1.2:

- Promovido a posição líder.

> - Requisito alterado: MSTG-STORAGE-1 "precisa de ser usado".

Reviewer: This is poiting to another document that is not translated, to we should keep the original name

```suggestion
- Requisito alterado: MSTG-STORAGE-1 "need to be used".
```

- Requisito MSTG-CODE-4 foi atualizado para suportar mais do que somente depuração.
- Requisito MSTG-PLATFORM-10 adicionado para mais segurança no uso de vistas web.
- Requisito MSTG-AUTH-12 adicionado para relembrar os programadores de reforçar a implementação de autorizações, especialmente em casos de aplicações multi-utilizador.  
- Adicionada mais alguma descrição em como o MASVS deve ser usado dada uma avaliação de risco.
- Adicionada mais alguma descrição sobre conteúdo pago.
- Requisito MSTG-ARCH-11 adicionado para incluir a política de Divulgação Responsável para aplicações L2.
- Requisito MSTG-ARCH-12 adicionado para demonstrar aos desenvolvedores de aplicações, que leis internacionais de privacidade importantes devem ser cumpridas.
- Criado um estilo consistente para todas as referências em versão Inglesa.
- Requisito MSTG-PLATFORM-11 adicionado para combater a espionagem através de teclados de terceiros.
- Requisito MSTG-MSTG-RESILIENCE-13 adicionado para impedir a espionagem de uma aplicação.

## V1.1.4 - 4 de Julho de 2019 - Edição da Cimeira

As seguintes alterações fazem parte do lançamento da versão 1.1.4:

> - Corrigidos todos os problemas de remarcação.

Reviewer: ```suggestion
- Corrigidos todos os problemas de Markdown.
```

Markdown here as a language, so we shouldn't translate it

## V1.1.2 - 3 de Janeiro de 2019 - Patrocínio e internacionalização

As seguintes alterações fazem parte do lançamento da versão 1.1.2:

- Adicionados agradecimentos aos compradores do e-book.
- Adiciondos links de autenticação e links atualizados de falhas de autenticação, em falta na V4.
- Retificada a troca de 4.7 e 4.8 em Inglês.
- Primeiro lançamento de versão internacional!
  - Correções em traduções de Espanhol. A tradução encontra-se agora em sincronia com a de Inglês (1.1.2).
  - Correções na tradução para Russo. A tradução encontra-se agora em sincronia com a de Inglês (1.1.2).
  - Adicionado o primeiro lançamento de versão em Cinês (ZHTW), Francês, Alemão e Japonês!
- Documento simplificado para facilitar a tradução.
- Adicionadas instruções para lançamento de versões automatizado.

## V1.1.0 - 14 de Julho de 2018

As seguintes alterações fazem parte do lançamento da versão 1.1:

- O requisito 2.6 "A área de transferência é desativada em campos de texto que contêm informação sensível." foi remvido.
- O requisito 2.2 "Informação sensível não deve ser guardada fora do repositório da aplicação ou do sistema de armazenamento de credenciais implementado" foi adicionado.

> - O requisito 2.1 foi reformulado para "O sistema de armazenamento de credenciais implementado é usado de forma apropriada para guardar informação sensível, como informações de identificação pessoal (PII), credenciais de utilizador ou chaves de encriptação.".

Reviewer: These 3 requirements should have the exact same text as described in 0x07-V2-Data_Storage_and_Privacy_requirements
